README.md

```diff
@@ -6,73 +6,72 @@ https://firecloud.dsde-dev.broadinstitute.org/
 
 https://firecloud.dsde-staging.broadinstitute.org/
 
-## Getting Started
+## Technologies
 
-This project runs in a Docker container. Docker is available here:
+[ClojureScript](https://github.com/clojure/clojurescript) is used for the UI.
 
-https://www.docker.com/
+We use the [Leiningen](http://leiningen.org/) build tool to manage ClojureScript dependencies.
 
-[ClojureScript](https://github.com/clojure/clojurescript) is used for the UI.
-We use the [Leiningen](http://leiningen.org/) build tool.
 The code incorporates usage of [react-cljs](https://github.com/dmohs/react-cljs) which is 
 a ClojureScript wrapper for [React](https://facebook.github.io/react/).
 
-For authentication, you will need to create a Google web application client ID from here:
+Figwheel replaces the running JavaScript within the page so changes are visible without a browser reload. More information [here](https://github.com/bhauman/lein-figwheel). [This video](https://www.youtube.com/watch?v=j-kj2qwJa_E) gives some insight into the productivity gains available when using this technology (up to about 15:00 is all that is necessary).
+
+## Getting Started
+
+Start with a [docker](https://www.docker.com/) environment.
+
+You will need to create a Google web application client ID from here:
 
 https://console.developers.google.com/
 
-## Building
+This should include your docker host as an authorized JavaScript origin. By convention, we use `dhost` as the hostname, which requires an addition to your /etc/hosts file, e.g.,
 
-Build once:
 ```
-./script/dev/build-once.sh
+192.168.99.100 dhost
 ```
 
-Watch files and rebuild whenever a file is saved:
-```
-./script/dev/start-auto-build.sh
-```
+The IP address is returned by `docker-machine ip default`.
+
+Set your client ID in your environment:
 
-Watch files, rebuild, and reload changes into the running browser window:
 ```
-./script/dev/start-hot-reloader.sh
+export GOOGLE_CLIENT_ID='...'
 ```
 
-[figwheel](https://github.com/bhauman/lein-figwheel) is used to accomplish this. [This video](https://www.youtube.com/watch?v=j-kj2qwJa_E) gives some insight into the productivity gains available when using this technology (up to about 15:00 is all that is necessary).
+Start the server in docker:
 
-This can take around 20 seconds to completely start. When ready, it will display the following message:
 ```
-Prompt will show when figwheel connects to your application
+./script/dev/start-server.sh
```

Comment:
After running this line (and after the previous steps as well...), I see this in the docker console:

```
[Fri Sep 18 11:25:21.577192 2015] [core:warn] [pid 32:tid 140164930586496] AH00111: Config variable ${CLIENTID} is not defined
[Fri Sep 18 11:25:21.577328 2015] [core:warn] [pid 32:tid 140164930586496] AH00111: Config variable ${CLIENTSECRET} is not defined
[Fri Sep 18 11:25:21.577750 2015] [core:warn] [pid 32:tid 140164930586496] AH00111: Config variable ${OIDC_SCOPES} is not defined
[Fri Sep 18 11:25:21.578014 2015] [core:warn] [pid 32:tid 140164930586496] AH00111: Config variable ${OIDC_COOKIE} is not defined
[Fri Sep 18 11:25:21.578221 2015] [core:warn] [pid 32:tid 140164930586496] AH00111: Config variable ${CLIENTID} is not defined
[Fri Sep 18 11:25:21.578436 2015] [core:warn] [pid 32:tid 140164930586496] AH00111: Config variable ${CLIENTSECRET} is not defined
```